corehq/apps/fixtures/tests/test_couchsqlmigration.py

```diff
@@ -383,6 +383,57 @@ def test_migration_with_old_doc_format(self):
             [],
         )
 
+    def test_migration_race(self):
```

Comment:
I'm not thrilled that this test will be deleted when the lookup tables migration is completed, but the code its testing is part of the migration framework, which will live on after that. It seems like a lot of overhead to create test model classes to simulate an in-progress migration, but maybe it's worth it? Thoughts?

cc @orangejenny